src/main/java/me/karakelley/http/Response.java

```diff
@@ -17,17 +22,48 @@ public void setHeaders(String header, String value) {
     headers.put(header, value);
   }
 
-  public void setBody(String body) {
+  public void setBody(byte[] body) {
     this.body = body;
   }
 
-  public String getBody() {
+  public void setBody(String body) {
+    this.body = body.getBytes();
+  }
+
+  public byte[] getBody() {
     if (body != null) {
       return body;
-    } else return "";
+    } else return "".getBytes();
+  }
+
+  public byte[] deliver() {
```

Comment:
Does this method actually send the contents out? The name `deliver` suggests it does, but it does return a `byte[]` which makes me think that something else is probably actually doing the delivery (otherwise this could presumably be `void`). Also the body of this method seems like it's just doing some conversion.